lib/core/print.toit

```diff
@@ -2,14 +2,17 @@
 // Use of this source code is governed by an MIT-style license that can be
 // found in the lib/LICENSE file.
 
+import monitor
+import system.api.print show PrintService PrintServiceClient
+
 /**
 Prints the $message.
 
 /* TODO(florian): the following sentence will probably get stale soon. */
```

Comment:
Remove the TODO.
Now that the `stringify` is even in the function, it's probably stable.